app/backend/src/tests/test_auth.py

```diff
@@ -47,7 +47,7 @@ def test_UsernameValid(db):
         assert not auth_api.UsernameValid(auth_pb2.UsernameValidReq(username="")).valid
 
 
-def test_signup_incremental(db):
+def test_signup_incremental(db, fast_passwords):
```

Comment:
No need to include `fast_passwords` fixture here since it is in the autouse above.

Reply:
Good point, thanks